modules/administration/pages/tshoot-sync.adoc

```diff
@@ -45,6 +45,17 @@ RepoMDError: Cannot access repository. Maybe repository GPG keys are not importe
 To resolve the problem, you need to import the GPG key to {productname}.
 For more on importing GPG keys, see xref:administration:repo-metadata.adoc[].
 
+
+GPG Key Removal::
+Once GPG key for repository is imported using [literal]``spacewalk-repo-sync command``, and is no longer needed (for example if the the key was  compromised, or was used for testing purposes only), it can be removed using the command:
```

Comment:
I just saw some typos here. I would suggest:

```suggestion
Once a GPG key for repository has been manually imported using [literal]``spacewalk-repo-sync`` command, and this key is no longer needed (for example if the key was compromised, or was used for testing purposes only), it can be removed from the zypper RPM database used by [literal]``spacewalk-repo-sync`` using the following command:
```